### PR TITLE
Add Rails.env.remote? 🙂 

### PR DESCRIPTION
**Pull request by loqimean**
Combined checks into one method `remote?` like a `local?` method

Before:

```ruby
if Rails.env.production? || Rails.env.staging? || Rails.env.ci?
```

After:

```ruby
if Rails.env.remote?
```

### CODE REVIEW

File: activesupport/lib/active_support/environment_inquirer.rb [hunk 1]
```diff
@@ -30,5 +31,10 @@ def initialize(env)
     def local?
       in? LOCAL_ENVIRONMENTS
     end
+
+    # Returns true if we are in the production or staging environment etc.
```
Comment (loqimean) [inline activesupport/lib/active_support/environment_inquirer.rb:35]:
```suggestion
    # Returns true if we aren't in development or test.
```

**Comment by Faq** [comment]
Consider to apply this too? https://github.com/rails/rails/pull/46830

**Comment by loqimean** [comment]
> Consider to apply this too? #46830

what do you mean?

**Comment by loqimean** [comment]
@dhh what about this one?)

**Comment by rafaelfranca** [comment]
Thank you for the pull request, but a simple `!` would be enough.

```
if !Rails.env.local?
```

or even

```
unless Rails.env.local?
```

**Pull request closed without merging**